supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -9,6 +9,116 @@
 
 *See also*:
 
+// AMQ: Added "In Red Hat AMQ"
+[discrete]
+[[acceptor]]
+==== image:images/yes.png[yes] acceptor (noun)
+*Description*: In Red Hat AMQ, an acceptor defines the way a client can connect to a broker instance.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+// RHDS: Kept as is, not specific to RHDS, correct?
+[discrete]
+[[access-control-instruction]]
+==== image:images/yes.png[yes] access control instruction (noun)
+*Description*: In an LDAP directory, an access control instruction (ACI) grants or denies access to entries or attributes. Use "access control instruction" on the first usage and then the abbreviation "ACI" subsequently.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+// RHSSO: Kept as is
+[discrete]
+[[access-token]]
+==== image:images/yes.png[yes] access token
+*Description*: An access token is a token that can be provided as part of an HTTP request that grants access to the service being invoked on. This is part of the OpenID Connect and OAuth 2.0 specification.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+// OpenShift: Kept as is since it specified OCP already
+[discrete]
+[[action]]
+==== image:images/yes.png[yes] action (noun)
+*Description*: In the context of authorization in OpenShift Container Platform, an action consists of _project_, _verb_, and _resource_.
```

Comment:
```suggestion
*Description*: In OpenShift Container Platform, an action consists of _project_, _verb_, and _resource_.
```
In the interests of minimalism, is it ok to delete the extra text?

Reply:
I don't think I would remove it, because noting that it's related to authorization is probably important. But I do think that we could move it elsewhere into the sentence so that "In OpenShift Container Platform" is first.

Reply:
```suggestion
*Description*: In OpenShift Container Platform, an authorization action consists of _project_, _verb_, and _resource_.
```